### DISCUSSION TITLE
Crowdin Markdown table+pipe issue

### DISCUSSION THREAD

**Discussion by slorber** [facebook/docusaurus#7051] [General]
## 🐛 Bug Report

We use pipe in markdown tables to describe TS union types

```md
| `editUrl` | <code>string \| EditUrlFunction</code> | `undefined` | Base URL to edit your site. The final URL is computed by `editUrl + relativeDocPath`. Using a function allows more nuanced control for each file. Omitting this variable entirely will disable edit links. |
```

But this leads to weird results on Crowdin:

https://crowdin.com/translate/docusaurus-v2/94/en-fr?filter=basic&value=0

![image](https://user-images.githubusercontent.com/749374/134309503-1fca8f3c-0524-4547-866d-d484326b232b.png)

And display badly on localized sites:

https://docusaurus.io/fr/docs/next/api/plugins/@docusaurus/plugin-content-docs#configuration

![image](https://user-images.githubusercontent.com/749374/134309940-500f05c1-fc69-4817-bf1d-6a24b1890ffb.png)

We should find a way to find a syntax that Crowdin can understand, or ask their support for a solution.

This is a follow-up of:
- Report: https://crowdin.com/project/docusaurus-v2/discussions/13
- Fix attempt: https://github.com/facebook/docusaurus/pull/5539

**Comment by slorber** [comment]
The Crowdin team is on it, they could reproduce the bug

**Comment by forresst** [comment]
@slorber Do you have news from Crowdin?

**Comment by slorber** [comment]
hey, yes forgot but got an answer:

![image](https://user-images.githubusercontent.com/749374/148373885-3d797781-f4c3-4cdf-bf6f-67333907336e.png)

**Comment by Josh-Cena** [comment]
Indeed, we don't have `<small>` now 😄 But I don't know if they actually mean `<small>` or `<code>`...

**Comment by forresst** [comment]
It is true that Crowdin's answer is a bit ambiguous.

How can we proceed to test Crowdin's workaround?

**Comment by slorber** [comment]
maybe we revert some changes of the linked PR and see if it works better after merge?

**Comment by forresst** [comment]
For your information, I found a workaround in Crowdin while waiting for Crowdin's correction

You just have to copy the original English version with the `<code>` and `</code>` tags

![image](https://user-images.githubusercontent.com/163352/155320227-e623fde5-e6cb-4aa3-b99a-119e269ce0cf.png)

The rendering in french on docusaurus.io is [now correct](https://docusaurus.io/fr/docs/next/api/plugins/@docusaurus/plugin-content-docs#configuration) : 

![image](https://user-images.githubusercontent.com/163352/155321161-a1cc592a-57f7-41b1-9186-3f146121da47.png)

**Comment by Josh-Cena** [comment]
@forresst Could you elaborate on the "copy the original English version"? Do you mean manually checking out the original Markdown source? Because the "Copy Source" button still copies gibberish.

Re-typing the `\|` certainly works, because it's Markdown after all.

**Comment by forresst** [comment]
@Josh-Cena Sorry I wasn't very clear.

"copy the original English version" => checking out the original Markdown source

**Comment by slorber** [comment]
Not sure if it's still relevant but Crowdin reported fixing the problem

<img width="833" alt="image" src="https://user-images.githubusercontent.com/749374/162000594-cb5be54c-9a15-4ac2-8b37-9e262f6b91ff.png">
